process/block/shardblock.go

```diff
@@ -1043,6 +1043,7 @@ func (sp *shardProcessor) CommitBlock(
 		highestFinalBlockNonce,
 		lastCrossNotarizedHeader,
 		header,
+		sp.managedPeersHolder,
```

Comment:
you forgot to set the managedPeersHolder value and is always nil. Failed at system testnet.
Please add this line on L121:
```GO
managedPeersHolder: arguments.ManagedPeersHolder
```
something similar to what we have on metablock.go L137

Reply:
:man_facepalming: fixed in #5371